ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -788,11 +788,41 @@ export default class SelectHardware extends Component {
         ),
       },
       {
-        message: this.context.t('QRHardwareWalletSteps2Description'),
-      },
-      {
-        asset: 'qrcode-wallet-demo',
-        dimensions: { width: '225px', height: '75px' },
+        message: (
+          <>
+            <p className="hw-connect__QR-subtitle">
+              {this.context.t('QRHardwareWalletSteps2Description')}
+            </p>
+            <Button
+              className="hw-connect__external-btn-first"
+              variant={BUTTON_VARIANT.SECONDARY}
+              onClick={() => {
+                this.context.trackEvent({
+                  category: MetaMetricsEventCategory.Navigation,
+                  event: 'Clicked Ngrave Buy Now',
+                });
+                openWindow(HardwareAffiliateLinks.ngrave);
+              }}
+              data-testid="ngrave-brand-buy-now-btn"
+            >
+              {this.context.t('buyNow')}
+            </Button>
+            <Button
+              className="hw-connect__external-btn"
+              variant={BUTTON_VARIANT.SECONDARY}
+              onClick={() => {
+                this.context.trackEvent({
+                  category: MetaMetricsEventCategory.Navigation,
+                  event: 'Clicked Ngrave Learn more',
+                });
+                openWindow(HardwareAffiliateTutorialLinks.ngrave);
+              }}
+              data-testid="ngrave-brand-learn-more-btn"
+            >
```

Comment:
@AlexJupiter are these events relevant for tracking?

Reply:
Yeah these events will be useful. I don't think this is actually following our data schema, but they are they same format as all the other events on this page so I think it's fine for now.